src/index.ts

```diff
@@ -1 +1,74 @@
-console.log("yo");
+import { convertFilesWithJson, convertFilesWithRegex } from "./jsonToCsv";
+import {
+  csvDataPath,
+  jsonDataPath,
+  readDataFiles,
+  isJestRunning,
+} from "./utils";
+import { existsSync, mkdirSync } from "fs";
+
+const header = [
```

Comment:
Yeah, manually update would be necessary. I created dynamic preparation of header (first row in the csv file).

Reply:
But I still don't know if I should handle nested json structures because I will have headers like below:
name,country,friend
And fiends would be array of objects with for example name, age, gender key-values, then I don't know how to even parse it